helm_deploy/values-preprod.yaml

```diff
@@ -26,7 +26,7 @@ env:
   JAVA_OPTS: "-Xmx2048m"
   HMPPS_AUTH_BASE_URI: "https://sign-in-preprod.hmpps.service.justice.gov.uk/auth"
   HMPPS_AUTH_REDIRECT_BASE_URI: "https://calculate-journey-variable-payments-preprod.apps.live-1.cloud-platform.service.justice.gov.uk"
-  CRON_IMPORT_REPORTS: "0 0 4 * * ?"
+  CRON_IMPORT_REPORTS: "0 30 5 * * ?"
```

Comment:
Note these times are using Spring CRON (not standard CRON).  Spring being Spring it does it slightly differently and allows for seconds hence the extra 0 at the front.